dialogflow-app.js

```diff
@@ -962,10 +965,10 @@ class DialogflowApp extends AssistantApp {
       if (textToSpeech.speech) {
         // Convert SimpleResponse to RichResponse
         textToSpeech = this.buildRichResponse().addSimpleResponse(textToSpeech);
-      } else if (!(textToSpeech.items &&
+      } else if (!((textToSpeech.items &&
         textToSpeech.items[0] &&
-        textToSpeech.items[0].simpleResponse)) {
-        error('Invalid RichResponse. First item must be SimpleResponse');
+        textToSpeech.items[0].simpleResponse) || textToSpeech.event)) {
```

Comment:
This conditional is getting a bit complicated. Can this be refactored?

Reply:
yes. I mean if (!(a || b)) is not clear enough we can do (!a && !b) :)